kore/test/Test/Kore/Step/Simplification/Equals.hs

```diff
@@ -805,9 +808,13 @@ test_equalsSimplification_TermLike =
             in
                 testCase "[a] `concat` x /\\ [a, b] "
                     (assertTermEquals
-                        (Condition.fromSingleSubstitution
-                            (ElemVar x, Mock.builtinList [Mock.b])
-                        )
+                        Conditional
+                            { term = ()
+                            -- TODO(virgil): This sort should be listSort.
```

Comment:
Does this need to be fixed?

Reply:
Maybe, but fixing it properly would require fixing this in the entire simplifier code, as opposed to fixing it in `Simplifier/TermLike.hs`, as it's done now. This may be a large change in itself, so I'm hesitant to do it now. How about doing it in the next PR if you think it's worth having consistent output from the individual simplifiers?

Reply:
Sounds good!